crates/environ/src/vmoffsets.rs

```diff
@@ -466,6 +472,11 @@ impl<P: PtrSize> From<VMOffsetsFields<P>> for VMOffsets<P> {
                 ret.num_escaped_funcs,
                 ret.ptr.size_of_vm_func_ref(),
             ),
+            size(typed_continuations_store)
+                = ret.ptr.size(),
+            align(16), // TODO(dhil): This could probably be done more
```

Comment:
I don't quite understand what is happening here in terms of alignment. Adding this `align` after the `typed_continutiaton_store` pointer simply increases the size of the `VMContext`. Is that the intended effect?

Reply:
I believe the entire context needs to be 16-byte aligned.